src/main/java/com/bettercloud/vault/response/SealResponse.java

```diff
@@ -0,0 +1,55 @@
+package com.bettercloud.vault.response;
+
+import com.bettercloud.vault.json.*;
+import com.bettercloud.vault.rest.RestResponse;
+
+import java.io.UnsupportedEncodingException;
+
+/**
+ * This class is a container for the information returned by Vault in <code>v1/sys/*seal*</code>
+ * operations.
+ */
+public class SealResponse extends VaultResponse {
+    private Boolean sealed;
+    private Long t;
```

Comment:
I know the JSON calls these `t` and `n` but can we have better names in the client?